webapp/components/Comment.vue

```diff
@@ -132,6 +135,9 @@ export default {
     editCommentMenu(showMenu) {
       this.openEditCommentMenu = showMenu
     },
+    async updateComment(comment) {
```

Comment:
why is this `async` if there is no `await`?
Is this really the best way to handle this situation @Tirokk @roschaefer 
we are emitting an event from a grandchild to a parent component, which just emits an event to its parent(granparent of CommentForm) so that it can update the list??
Seems strange this would be the best way to do this....

Reply:
The `async` slipped in by copy and paste I suppose and I have overseen it. @mattwr18 
I removed it.

This @roschaefer suggested and I followed him, because I don’t know a better way yet. And we need something what works for sure for the moment.

May the `vue-states` + `vue-history` packages from our last Vue MeetUp would help here in the future?